x/auth/client/cli/tx_multisign.go

```diff
@@ -183,9 +183,11 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		defer fp.Close()
+		if err = fp.Close(); err != nil {
```

Comment:
Same problem here. If `Close()` fails then `PrintBytes(json)` does not get executed.